R/compare.R

```diff
@@ -0,0 +1,42 @@
+#' Solve different single P pool MEND configurations
+#'
```

Comment:
@jianqiuz right now the MEND and MEND2 models are compared in two separate functions. Is this okay or would we want to do the comparison in a single function? I think that would mean we would probably need to aggregate the 2 pool MEND model pools to compare the results with the MEND results. Would we need to change the parameter values to make them consistent with one another?

Reply:
One idea I have been bouncing around is having a single function with multiple pools. We get the one pool structure by setting kinetic parameters at the same value for all pools.